bundle/manifests/numaresources-operator.clusterserviceversion.yaml

```diff
@@ -256,13 +256,13 @@ spec:
         serviceAccountName: numaresources-controller-manager
     strategy: deployment
   installModes:
-  - supported: false
+  - supported: true
```

Comment:
please elaborate in the commit message about why this change is needed

Reply:
done